app/src/main/res/layout/story_chapter_view.xml

```diff
@@ -108,6 +108,8 @@
             android:layout_marginTop="8dp"
             android:layout_marginEnd="8dp"
             android:layout_marginBottom="4dp"
```

Comment:
All all the views where you have used `minHeight` / `minWidth`, if that view contains "margin", shift that to padding.
For example for this view you can remove
```
            android:layout_marginStart="8dp"
            android:layout_marginTop="8dp"
            android:layout_marginEnd="8dp"
            android:layout_marginBottom="4dp"
```

and replace with
```
            android:paddingStart="8dp"
            android:paddingTop="8dp"
            android:paddingEnd="8dp"
            android:paddingBottom="4dp"
```

Reply:
Done